crates/db_views/src/structs.rs

```diff
@@ -85,7 +85,6 @@ pub struct PostView {
   pub read: bool,                 // Left join to PostRead
   pub creator_blocked: bool,      // Left join to PersonBlock
   pub my_vote: Option<i16>,       // Left join to PostLike
-  pub language: Language,
```

Comment:
The front ends already get back a list of languages and their codes with the site, so this isn't necessary.